scss/utilities/_json-api.scss

```diff
@@ -66,44 +71,40 @@
 /// ### Colors
 ///
 /// Each color has an access name and value.
-/// For us, that is often a map-key string,
-/// and Sass hex/rgba output color:
+/// The key will be used to identify the correct data
+/// for a given color-palette,
+/// and the data includes color name:value pairs,
```

Comment:
add a space between `name:` and `value pairs`?

Reply:
![screen shot 2017-11-27 at 1 16 30 pm](https://user-images.githubusercontent.com/1581694/33284649-63b93c38-d375-11e7-81c0-515936a92baa.jpg)

Reply:
well, the ":" isn't acting like it would in a sentence. I think we could do either "name/value pairs" or as code "`name: value` pairs"?

Reply:
(you can also make this change if you have a chance)